Documentation/trying-out-rkt.md

```diff
@@ -22,6 +22,12 @@ cd rkt-v1.0.0
 
 To faciliate privilege separation between different commands, rkt uses a `rkt` group that has read-write access to the rkt data directory. This allows `rkt fetch`, which downloads and verifies images, to run as a non-root user. If you skip this section, you can run `sudo rkt fetch` instead but we recommend taking the following steps for production use cases.
 
+First, trust the signing key for etcd images. This must be run as root as access to the keystore is restricted:
+
+```
+sudo ./rkt trust --prefix coreos.com/etcd
+```
```

Comment:
The decision to move the trust command up was because the paragraph started with "First, ..." and is always a mandatory step anyway. I don't mind moving it back.